app/src/app/home/home.component.sass

```diff
@@ -14,10 +14,11 @@ a
     text-decoration: underline
 
 div#logos
-  display: flex
+  display: inline-flex
   margin-top: 2em
-  justify-content: center
   align-items: center
+  justify-content: space-around
+  align-content: center
```

Comment:
align-content has no effect because by default flex-wrap is nowrap, hence the flex has only one line.